pom.xml

```diff
@@ -28,6 +28,15 @@
     </dependencies>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>3.1.1</version>
+                <configuration>
+                    <configLocation>my-checkstyle-checker.xml</configLocation>
+                </configuration>
+            </plugin>
+
```

Comment:
Why did you add these configurations?

Reply:
At the first practice, I thought that it is a part of the project but then I understood that it is a general setting)